pkg/apis/pxc/v1/pxc_types.go

```diff
@@ -28,8 +28,17 @@ type PerconaXtraDBClusterSpec struct {
 	Backup                *PXCScheduledBackup                  `json:"backup,omitempty"`
 	UpdateStrategy        appsv1.StatefulSetUpdateStrategyType `json:"updateStrategy,omitempty"`
 	AllowUnsafeConfig     bool                                 `json:"allowUnsafeConfigurations,omitempty"`
+	enableSmartUpdate     bool
```

Comment:
can we avoid using this flag?

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/369/commits/212814f5bbccb878d987e3a017e3dc72e57d19d0